yt_dlp/extractor/iprima.py

```diff
@@ -136,7 +254,7 @@ def extract_formats(format_url, format_key=None, lang=None):
                 extract_formats(src)
 
         if not formats and '>GEO_IP_NOT_ALLOWED<' in playerpage:
-            self.raise_geo_restricted(countries=['CZ'], metadata_available=True)
+            self.raise_geo_restricted(countries=['CZ'])
```

Comment:
why remove metadata_available? It allows user to extract the metadata with `--ignore-no-formats-error` even if geo-blocked

Reply:
My bad, I was originally writing this for youtube-dl, which does not allow this in their `raise_geo_restricted`. I will something similar to this to the other extractor as well.